src/App.vue

```diff
@@ -234,15 +232,9 @@ import { SessionStorageKeys } from 'sbc-common-components/src/util/constants'
     WebChat,
     ...Dialogs,
     ...Views
-  },
-  mixins: [
-    CommonMixin,
-    DateMixin,
-    FilingTemplateMixin,
-    NameRequestMixin
-  ]
+  }
 })
-export default class App extends Vue {
+export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMixin, NameRequestMixin) {
```

Comment:
Does this change for using Mixins have to happen for the other repos as well Sev?

Reply:
Yes, but I was going to make them in a future ticket.

History: in originally thought that for better Vue3 compatibility, we should import mixins using the options syntax, but:
- that broke typings (and displayed build errors, even though it did work)
- there is a similar class decorator for Vue3 after all

Reply:
Oh okay, makes sense. Thanks for the explanation!

I was thinking to make this change in Filings UI since I'm already in the process of changing all the files for Pinia. But I'm just going to focus on Pinia now and we'll fix that in a later ticket like you said.

Reply:
I got into this to help Ketaki focus on Pinia only (and that only worked so-so).

The catch is that you'll have to fix a bunch of files (as in this PR) to pre-empt mixin errors and the like.

On the other hand, do you really want me to make these changes to Filings UI and require you to rebase? I don't have a sense of whether auto-merge will work well or not 🤷‍♂️

Reply:
Yes, I figured as much. No Sev, I'll try to fix the files and we'll go from there. I think rebasing would be a nightmare If I try to do it after the same changes end up being done in Filings UI.